docs/ecosystem/community/zio-temporal.md

```diff
@@ -0,0 +1,153 @@
+---
+id: zio-temporal
+title: "ZIO Temporal"
+---
+
+[ZIO Temporal](https://zio-temporal.vhonta.dev/) is a ZIO library for Temporal, a microservice workflow orchestration platform.
+
+## Introduction
+
+ZIO Temporal is a ZIO library based on the Temporal Java-SDK. [Temporal](https://temporal.io/) helps to eliminate complex error or retry logic, avoid callbacks, and ensure that every workflow you start, completes. Temporal delivers durable execution for your services and applications.
+
```

Comment:
Let's add one more sentence here:

`ZIO Temporal brings first-class Scala & ZIO support with additional compile-time checks that Java SDK lacks`